mappings/net/minecraft/server/network/ServerPlayNetworkHandler.mapping

```diff
@@ -71,3 +71,13 @@ CLASS net/minecraft/class_3244 net/minecraft/server/network/ServerPlayNetworkHan
 		COMMENT <p>For this to return true, the player must not be actively cooling down.
 		ARG 0 player
 		ARG 1 stack
+	METHOD method_31275 filter (Ljava/lang/Object;Ljava/util/function/Consumer;Ljava/util/function/BiFunction;)V
+		ARG 1 message
+		ARG 2 consumer
+		ARG 3 backingFilterer
+	METHOD method_31277 filterString (Ljava/lang/String;Ljava/util/function/Consumer;)V
```

Comment:
questionable name. maybe call it filterMessage for context?

Reply:
Thing is that it's not just messages, I figured that out in the second commit. It's also used for books and renaming items with an anvil

Reply:
Would be better to name the classes differently then, to include the fact it's used by books and anvils too

Reply:
Renamed to `filterText`